src/test/ProposalPayloadE2E.t.sol

```diff
@@ -30,7 +30,10 @@ contract ProposalPayloadE2ETest is Test {
 
     function testExecute() public {
         // Pre-execution assertations
-
+        // I'm having issues what to even test here. if the deployment of the new interest rate strategy is correct,
```

Comment:
Test the new strategy address
Test the new strategy values like we did for wMATIC

Reply:
do you know any easy way to test those as we did for v3?

couldn't find the equivalent for v2